SIS_import/sis_set_url.rb

```diff
@@ -529,24 +504,26 @@ def read_argv
 		process_error = "Cannot find logs directory " + outputDirectory
 	else
 		begin
-			outputFile = File.open(outputDirectory + "Canvas_set_url_#{Time.new.strftime("%Y%m%d%H%M%S")}.txt", "w")
+			outputFileName = outputDirectory + "Canvas_set_url_#{Time.new.strftime("%Y%m%d%H%M%S")}.txt"
+			outputFile = File.open(outputFileName, "w")
```

Comment:
The logger has been pre-initialized to point to STOUT. If the log file can be open, the logger will write messages into the log file; otherwise, logger continues to write to STOUT.